src/timesync/timesyncd-bus.c

```diff
@@ -163,6 +244,7 @@ static const sd_bus_vtable manager_vtable[] = {
         SD_BUS_PROPERTY("LinkNTPServers", "as", property_get_servers, offsetof(Manager, link_servers), 0),
         SD_BUS_PROPERTY("SystemNTPServers", "as", property_get_servers, offsetof(Manager, system_servers), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("FallbackNTPServers", "as", property_get_servers, offsetof(Manager, fallback_servers), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_WRITABLE_PROPERTY("RuntimeNTPServers", "as", property_get_runtime_servers, property_set_runtime_servers, 0, 0),
```

Comment:
as mentioned, please make this  read-only prop, and instead add a SetRuntimeNTPServers() method for changing them.